### PR TITLE
Deploy 21.11

### PR DESCRIPTION
**Pull request by nashthecoder**
[MVC-1181] - Remove ecosystem section from the website 
Currently investing issue with conflicts in merge. 

EDIT: Further US from commits:
[MVC-956] Refactored header - Tested autoplay on scroll function
[MVC-988] Website Responsiveness: Text adaptation to resolution
[MVC-1161] - Refactor header 
[MVC-1116] - Join Us - Refactor header
[MVC-1181] - Removed ecosystem section from the website

[MVC-1181]: https://triggerise.atlassian.net/browse/MVC-1181
[MVC-956]: https://triggerise.atlassian.net/browse/MVC-956
[MVC-988]: https://triggerise.atlassian.net/browse/MVC-988
[MVC-1116]: https://triggerise.atlassian.net/browse/MVC-1116
[MVC-1181]: https://triggerise.atlassian.net/browse/MVC-1181

### CODE REVIEW

**Comment by raguiar9080** [comment]
@nashthecoder From a bit of investigating, the first commit that diverges the history is: https://github.com/Triggerise/triggerise.github.io/commit/225ff70cf7a4d050e25fb356552dad8d7e6663ee

I believe the issue here is that changes on master are not being sent to develop so we're seeing conflicts on things that were changed in develop and also master in the meantime. So probably for this the best way is to deal with the conflicts on a manual basis. I think this first time merging will be the most troublesome since a lot of conflicts come from a long way back. I can help with dealing with the merges if needed 👍

**Comment by nashthecoder** [comment]
Ok. I will review these files one by one work through it. Incase of a hitch , I will reach out.

**Comment by nashthecoder** [comment]
@raguiar9080 Seems as there are no more conflicts.

File: _includes/footer.html [hunk 1]
```diff
@@ -2,8 +2,8 @@
   <div class="wrapper">
     <div class="flex">
       <div class="footer-left">
-        <h2 class="footer-heading">{% include logo.html %}</h2>
-        <h3>Connecting for change</h3>
+        <!--  <h2 class="footer-heading">{% include logo.html %}</h2>  -->
```
Comment (raguiar9080) [inline _includes/footer.html:5]: This commented line seems like it can be removed ✂️

**Review by raguiar9080 — APPROVED** [review]
Looks good @nashthecoder 👍 
Left a comment regarding a commented line, but it can probably move as is and have it removed in a clean up PR sent to develop

**Comment by nashthecoder** [comment]
Closing pull request since update on Siteleaf merged develop branch to master.

**Pull request closed without merging**